inotify.go

```diff
@@ -88,31 +88,46 @@ func (w *Watcher) Close() error {
 
 // Add starts watching the named file or directory (non-recursively).
 func (w *Watcher) Add(name string) error {
-	name = filepath.Clean(name)
+	return w.AddFlags(name, Create|Remove|Write|Rename|Chmod)
+}
+
+// Add a watch limited to the particular operations
+func (w *Watcher) AddFlags(name string, flags Op) error {
+        name = filepath.Clean(name)
 	if w.isClosed() {
 		return errors.New("inotify instance already closed")
 	}
+	var inotifyFlags uint32
```

Comment:
If we name the function argument fsnotifyFlags (or something similar), and the inotifyFlags variable 'flags' we'll generate a much cleaner diff.